spec/features/cases/offender_sar/add_data_requests_spec.rb

```diff
@@ -82,16 +82,44 @@
     expect(data_request_page).to have_text 'errors prevented this form'
   end
 
-  scenario 'no data entry fails' do
+  scenario 'no data entry fails', js: true do
     cases_show_page.load(id: offender_sar_case.id)
     click_on 'Record data request'
 
     data_request_page.form.location.fill_in(with: '    ')
-    data_request_page.form.request_type.fill_in(with: '')
 
     click_on 'Continue'
 
     expect(data_request_page).to be_displayed
     expect(data_request_page).to have_text '3 errors prevented this form from being submitted'
   end
+
+  scenario 'record data request with data type of NOMIS other and notes', js: true do
+    request_values = {
+      location: 'HMP testing',
+      request_type: 'all_prison_records',
+      request_type_note: 'Testing nomis-other note',
+      date_requested: Date.new(2020, 8, 15),
+      date_from: Date.new(2018, 8, 15),
+      date_to: Date.new(2019, 8, 15),
+    }
+
+    cases_show_page.load(id: offender_sar_case.id)
+    click_on 'Record data request'
+
+    data_request_page.form.location.fill_in(with: request_values[:location])
```

Comment:
Might read slightly cleaner to encapsulate these blocks in methods. At least it would be easier to parse :)